openstack_controller/datadog_checks/openstack_controller/api.py

```diff
@@ -100,6 +100,11 @@ def get_project_limits(self, tenant_id):
         limits = server_stats.get('limits', {}).get('absolute', {})
         return limits
 
+    def get_flavors_detail(self, query_params, timeout=None):
+        url = '{}/flavors/detail'.format(self.endpoint)
+        flavors = self._make_request(url, self.headers, params=query_params, timeout=timeout)
+        return flavors.get('flavors', [])
```

Comment:
`flavors` can be `None` we should check for it

Reply:
hum, how could it be `None`?

Reply:
Scratch it, was looking at https://github.com/DataDog/integrations-core/blob/5fffc5bfcf2b30ef27a5e36388137a6fee34c645/openstack_controller/datadog_checks/openstack_controller/api.py#L41 but didn't notice the condition before